client/my-sites/checkout/checkout/composite-checkout-thank-you.ts

```diff
@@ -128,17 +124,6 @@ export function getThankYouPageUrl( {
 		return `${ newBlogUrl }/${ pendingOrReceiptId }`;
 	}
 
-	const redirectPathForGSuiteUpsell = getRedirectUrlForGSuiteNudge( {
-		pendingOrReceiptId,
-		siteSlug,
-		cart,
-		didPurchaseFail,
-		isNewlyCreatedSite,
```

Comment:
@sirbrillig, @nbloomf : `didPurchaseFail` and `isNewlyCreatedSite` are no longer used in this component, should we remove them?